pkgs/development/python-modules/hightime/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  setuptools,
+  pytestCheckHook,
+  pytest-cov,
+}:
+
+buildPythonPackage rec {
+  pname = "hightime";
+  version = "0.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ni";
+    repo = "hightime";
+    rev = "v${version}";
+    hash = "sha256-P/ZP5smKyNg18YGYWpm/57YGFY3MrX1UIVDU5RsF+rA=";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  build-system = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov
```

Comment:
**[nixpkgs-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>pytest-cov in nativeCheckInputs - remove this from nativeCheckInputs or change to pytest-cov-stub
<!-- __reviewdog__:ChBkYmEzNWFmZmQyZTJkYTMzEgxuaXhwa2dzLWxpbnQ= -->